packages/carbon-graphs/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * Changed
   * Fixed linter issues.
+  * Fixed Y-axis range during reflow. 
```

Comment:
Nit: Fixed Y-axis range *bug in* reflow.

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/66/commits/c46464886605a86eea3e0af75fa9d5f26fab19e5#diff-7b73f4b73c95577a16e8a58d2e1377b2204811d315c020884e80b9f22128c42aR7).